keps/sig-node/2008-forensic-container-checkpointing/README.md

```diff
@@ -106,13 +107,24 @@ For the first implementation we do not want to support restore in the
 outside of Kubernetes. The restore is a container engine only operation
 in this first step.
 
-The forensic use case is targeted to be part of the next (1.24) release.
+#### CRI Updates
 
-Although this KEP only adds checkpointing support to the kubelet the CRI API in
-the corresponding code pull request is extended to support *checkpoint* and
-*restore* in the CRI API. The reason to add *restore* to the CRI API without
-implementing it in the kubelet is to make development and especially testing
-easier on the container engine level.
+The CRI API will be extended to introduce one new RPC:
+```
+    // CheckpointContainer checkpoints a container
+    rpc CheckpointContainer(CheckpointContainerRequest) returns (CheckpointContainerResponse) {}
+```
+with the following parameters:
+```
+message CheckpointContainerRequest {
```

Comment:
needs a security statement / set of requirements for this service, validating the request and destination are authorized.